### ISSUE TITLE
Need policy on discounts/promo codes topic papers by multiple authors

### ISSUE THREAD

**Issue by ChristopherA** [WebOfTrustInfo/rwot11-the-hague#44]
We've had submissions of topic papers by multiple people in the past, and we gave multiple discount codes. However, in all of these cases, the papers were comprehensive and all the authors significantly contributed.

However, we've never had a formal policy about this, however, the submission of [eSSIF-Lab: Towards a European SSI ecosystem](https://github.com/WebOfTrustInfo/rwot11-the-hague/blob/master/advance-readings/eSSIF-Lab%20-%20Towards%20a%20European%20SSI%20ecosystem.md) pushes us to address it. First, for the authors of this specific paper, and in general for the future.

My proposal is that list of authors must be explicit and should have no more than 3 authors; that the submission needs closure (no rolling PRs) to include all the authors before any promo codes are offered, and that an RWOT Board Member can exceptions for significant collaborations.

Comments?

**Comment by RieksJ** [comment]
I don't argue that the submission in question might present the urge to come with a (formal) policy. However, presenting a policy and immediately enforcing it is like changing the rules in the course of the game. I do not know the actual consequences of this, but there might be a risk that people that have not attended RWOT earlier on may no longer have the will (or the possibility, as the discount is significant) to participate. In the case of eSSIF-Lab subgrantees, that have done good work in the SSI arena, that might be a loss.

Regarding the submission itself: I think it is a nice idea to have contributions of various SMEs that have not only worked on their own topic (which would have warranted an individual paper proposal), but in particular have also worked together (interop), and have shown they use each other's stuff,  in a single paper. Obviously, there are many ways in which that could be embodied, and the submission's author has presented a perhaps somewhat unconventional form, but that might be something to be discussed with the author(s).

In RWOT, participants are invited to work on papers, but that does not imply that work on every proposed paper gets done: people might find other papers more interesting/relevant to work for. This is a characteristic that helps augment the quality and relevance of such papers. The submission you refer to seems to attempt to take this one level further, by attempting to organize this up-front. Why not give this a chance, see how/if it works, and from the actual experience draft a policy if that is deemed appropriate.

**Comment by Alexjblom** [comment]
I completely agree with Rieks, presenting a policy and immediately enforcing it while as I understand

- you were made aware of the plans for submission by eSSIF-Lab some 6 months ago 
- and failed to directly communicate with SSIF-Lab about revising the policy

does not sound like a great idea.  eSSIF-Lab is even a sponsor of yours, correct?

**Comment by jandrieu** [comment]
This is not a change in policy.

Our policy has always been that substantive contributions to the workshop, in the form of an advance reading paper, deserve a discount.

We appreciate that the nuance of that policy may not have been apparent and we are already updating the documentation to make the policy clearer.

However, at no point did we ever consider allowing an open-ended single paper submission for all eSSIF participants. We have tried to be clear that eSSIF participants are welcome to participate in the workshop, just like all our participants. Submit a paper, get a discount.

It was never our intention, nor have we ever implied, that eSSIF participants who are unwilling to write an advance reading paper would get a discount. RWOT just doesn't work that way. What we need at the workshop are people who, in fact, are willing to write in a collaborative fashion.

To @Alexjblom's point: this has nothing to do with eSSIF as a sponsor. Sponsors get one or two comps to the event, depending on their level of support. The level of support given did not--and probably never could--justify extending the discount for a proven willingness-to-write to an arbitrary large number of the sponsor's allies.

It's unfortunate that we had this misunderstanding, but I want to assure you that this is the result of long-standing policy that has applied to every single RWOT event I have been apart of. Since I only missed RWOT I, I am confident in saying that this is a core part of how RWOT does RWOT.

**Comment by peacekeeper** [comment]
I agree with @jandrieu , I think the current eSSIF-Lab paper is interesting, but it doesn't feel like it contains the "equivalent" of individual topic paper submissions. The sections in that current paper are mostly self-promoting summaries of existing projects without clear connections between them (other than being part of eSSIF-Lab).

In my experience, a topic paper should be about some problem or new idea to be further elaborated on at RWOT. Writing a topic paper doesn't take long and it doesn't have to have a lot of content, but traditionally it has been something with individual new effort and thinking behind it.

I think discounts for topic papers are still be available now, even after the "early bird" period is over, or..?

**Comment by jandrieu** [comment]
@peacekeeper Yes, there are still discounts for advance readings, through Aug 26th (for purchase) with a paper deadline of Aug 19.

In addition, we are also extending the early bird deadline a bit for eSSIF grantees since it took some time to clarify the situation. Hopefully that will allow everyone at eSSIF who wants to participate to get a reasonable discount.
